rep-I0004.rst

```diff
@@ -295,8 +295,10 @@ Motoman
       2009  ROS_MSG_MOTO_WRITE_IO_GROUP           Write whole group (byte) of 'Network Inputs'
       2010  ROS_MSG_MOTO_WRITE_IO_GROUP_REPLY     -
       2011  ROS_MSG_MOTO_IOCTRL_REPLY             -
```

Comment:
What is this message ID for btw?

It was added in #26, but there is no description. There is also no corresponding request msg ID.

Reply:
There is a bit of code referencing it in MotoROS ([here](https://github.com/ros-industrial/motoman/blob/f4fb63984061f8fca2a9a3f886a41137a1ee0764/motoman_driver/MotoPlus/SimpleMessage.h#L331-L336) fi). But that doesn't appear to be actually used other than [here](https://github.com/ros-industrial/motoman/blob/f4fb63984061f8fca2a9a3f886a41137a1ee0764/motoman_driver/MotoPlus/SimpleMessage.h#L370).

Reply:
So... It looks like this message was intended to be used here. But a bad copy/paste was missed.
https://github.com/ros-industrial/motoman/blob/4b0a030de3917aa13ab1407f2dcf131d4f0ffd27/motoman_driver/MotoPlus/SimpleMessage.c#L173-L182

I feel like this should be updated, but I don't want to break existing code.
Thoughts?

Reply:
Seems it's only ever used in the IO server, which I doubt has too many users at the moment. And only if clients send a malformed or incorrect service request will they receive the result of calling `Ros_SimpleMsg_IoReply(..)`.

I would suggest to fix it, but let's do it as part of a separate PR (ie: not in https://github.com/ros-industrial/motoman/pull/369). Let's get https://github.com/ros-industrial/motoman/pull/369 merged first.

Reply:
**But**: there should be no need for a separate `ROS_MSG_MOTO_IOCTRL_REPLY`, and it also shouldn't work: replies should be sent back with the same message IDs as their requests, especially in case of errors.